phpBB/includes/functions.php

```diff
@@ -3648,26 +3647,19 @@ function phpbb_checkdnsrr($host, $type = 'MX')
 	// checkdnsrr() supports AAAA since 5.0.0
 	// checkdnsrr() supports TXT since 5.2.4
 	if (
-		($type == 'MX' || DIRECTORY_SEPARATOR != '\\' || version_compare(PHP_VERSION, '5.3.3', '>=')) &&
-		($type != 'AAAA' || version_compare(PHP_VERSION, '5.0.0', '>=')) &&
-		($type != 'TXT' || version_compare(PHP_VERSION, '5.2.4', '>=')) &&
-		function_exists('checkdnsrr')
+		($type == 'MX' || DIRECTORY_SEPARATOR != '\\') &&
+		$type != 'AAAA' && $type != 'TXT' && function_exists('checkdnsrr')
```

Comment:
I think I change this too quickly.
Checking if the `checkdnsrr ` function exists is enough, no? I did not find information saying if it can be disabled is some configuration of PHP.